github/ci/prow/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-presubmits-0.42.yaml

```diff
@@ -59,6 +59,35 @@ presubmits:
               - "/bin/sh"
               - "-c"
               - "automation/check-patch.e2e-workflow-k8s.sh"
+    - name: pull-cluster-network-addons-operator-unit-test
+      branches:
+        - release-0.42
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      max_concurrency: 6
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: bare-metal-external
+        containers:
+          - image: kubevirtci/bootstrap:v20201119-a5880e0
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
```

Comment:
Do you need so much memory for unit tests? I guess you didn't have that much in travis, maybe better start with less (4Gi?) and adapt if needed, wdyt?

Reply:
I agree 4Gi would be more than enough.